app/src/main/res/layout/activity_signup.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context="com.veephealthoutloud.healthoutloud.SignupActivity">
+
+
+    <TextView
+        android:id="@+id/textView2"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="User Sign-Up"
+        android:textSize="25sp"
+        tools:layout_editor_absoluteX="52dp"
+        tools:layout_editor_absoluteY="107dp" />
+
+    <TextView
+        android:id="@+id/textView"
+        android:layout_width="67dp"
+        android:layout_height="22dp"
+        android:text="Email:"
+        tools:layout_editor_absoluteX="52dp"
+        tools:layout_editor_absoluteY="175dp" />
+
+    <EditText
+        android:id="@+id/editText5"
```

Comment:
you should set the ids of the EditTexts so that they make sense when you use them later in the code. ex. "@+id/signup_email_textview"